bootstrap-notify/build.boot

```diff
@@ -0,0 +1,32 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
```

Comment:
Based on library bower.json, this library depends on jquery and bootstrap: https://github.com/mouse0270/bootstrap-notify/blob/master/bower.json
This package should thus depend on cljsjs/jquery and cljsjs/bootstrap